src/main/java/appeng/hotkeys/InventoryLocatingService.java

```diff
@@ -0,0 +1,44 @@
+package appeng.hotkeys;
+
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+
+import appeng.api.hotkeys.LocatingService;
+
+public class InventoryLocatingService implements LocatingService {
+
+    private final Locatable locatable;
+    private final Opener opener;
+
+    public InventoryLocatingService(Locatable locatable, Opener opener) {
+        this.locatable = locatable;
+        this.opener = opener;
+    }
+
+    public InventoryLocatingService(Item item, Opener opener) {
+        this((stack) -> stack.getItem() == item, opener);
+    }
+
+    @Override
+    public boolean findAndOpen(Player player) {
```

Comment:
We don't necessarily need to support them, but I'm curious how this would work with trinkets, if at all? I suspect trinket support could be added by an addon.